src/WallEvaluatorColloid.h

```diff
@@ -52,8 +60,52 @@ namespace detail
 class WallEvaluatorColloid
     {
     public:
-    //! Define the parameter type used by this wall potential evaluator
-    typedef Scalar2 param_type;
+    struct param_type
```

Comment:
There is going to be alot of duplication of boilerplate code between this and the LJ93 class. Please make a `WallEvaluator.h` file that implements a `WallParameters` and `WallEvaluator` base classes that set sensible defaults. See:
https://github.com/mphowardlab/azplugins/blob/azplugins-v1/src/PairEvaluator.h
for an example.